docs/docs/getting-started.server.mdx

```diff
@@ -506,7 +506,47 @@ Which in turn lets us choose whether to use the `@mdx-js/mdx` or
 through webpack loader syntax in imports.
 
 Install the webpack loader [`@mdx-js/loader`][mdx-loader].
-There is no need to configure it.
+
+To enable direct MDX imports w/o `!@mdx-js/loader!` prefix, the loader can
+further be added to webpack’s config, using a
+[`react-scripts`](http://github.com/facebook/create-react-app/tree/main/packages/react-scripts)
+rewiring tool, e.g. [CRACO](http://github.com/gsoft-inc/craco).
+
+<details>
+  <summary>Expand CRACO example</summary>
+
+  ```js path="craco.config.js"
+  const { addAfterLoader, loaderByName } = require('@craco/craco')
+
+  module.exports = {
+    webpack: {
+      configure: (webpackConfig) => {
+        addAfterLoader(webpackConfig, loaderByName('babel-loader'), {
+          test: /\.(md|mdx)$/,
+          loader: require.resolve('@mdx-js/loader')
+        })
+
+        return webpackConfig
+      }
+    }
+  }
+  ```
+
+  ```jsx path="src/App.jsx"
+  import Content from './content.mdx'
+
+  export default function App() {
+    return <Content />
+  }
+  ```
+</details>
+
+<Note type="info">
+  **Note**: due to [broken MDX import forwarding](http://github.com/facebook/create-react-app/issues/12166)
+  in `react-scripts` 5.x, rewiring is currently necessary when using CRA 5.
+  There’s an [ongoing discussion](http://github.com/mdx-js/mdx/discussions/1870)
+  about this issue.
```

Comment:
```suggestion
  **Note**: rewiring is currently needed for CRA 5,
  due to a bug in `react-scripts`
  ([`facebook/create-react-app#12166`](http://github.com/facebook/create-react-app/issues/12166)),
  which is also tracked at
  [`mdx-js/mdx#1870`](http://github.com/mdx-js/mdx/discussions/1870).
```

How about reworded like this? Puts what it means for users (have to rewire) at the front. Then explains and links to in-depth stuff.